### PR TITLE
Test refactor: bad_input.js

### PR DESCRIPTION
**Pull request by ccowell**

### CODE REVIEW

File: test/helpers/web3/initializeTestServer.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const Ganache = require(process.env.TEST_BUILD
+  ? "../../../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+const Web3 = require("web3");
+
+const intiailizeTestServer = (tests, port = 12345) => {
```
Comment (davidmurdoch) [inline test/helpers/web3/initializeTestServer.js]: I don't think this file should be here. Also, `initialize` is spelled wrong. :-D

File: test/vm.js [hunk 2]
```diff
@@ -1,74 +1,39 @@
...
-
-var web3 = new Web3();
-web3.setProvider(
-  Ganache.provider({
-    /* blockTime: 100, */
-    logger: logger,
-    seed: "1337"
-  })
-);
+const assert = require("assert");
```
Comment (davidmurdoch) [inline test/vm.js]: Is this file supposed to be here?

Comment (ccowell) [reply test/vm.js]: No. I'll restore from `develop` in the next commit.

File: test/helpers/web3/initializeTestServer.js (hunk 1 above)
Comment (ccowell) [reply test/helpers/web3/initializeTestServer.js]: Where do you think it should go?

**Pull request closed without merging**